resources/application-connector/values.yaml

```diff
@@ -87,5 +87,7 @@ tests:
     skipSslVerify: true
     image:
       dir:
+      # Prow job 'pre-kyma-tests-application-connector-tests" doesn't get
```

Comment:
Does a `/test pre-kyma-tests-application-connector-tests` help in order to create the image?

Reply:
To trigger the prow job manually from within a PR.

Reply:
I did that, but it will only create another (identical) PR image.

Reply:
True, normally only the post submit jobs create a real sha.

Reply:
It doesn't for this image, hence my comment.

Reply:
`post-kyma-tests-application-connector-tests` should do we trick, but you are right, it didn't run. At least I don't see it on `https://status.build.kyma-project.io/?type=postsubmit&job=post-kyma-tests-application-connector-tests`